docs/tutorials/screen1d.rst

```diff
@@ -0,0 +1,372 @@
+************************
+Using the Screen1D class
+************************
+
+This tutorial describes how to generate synthetic data corresponding to a
+single-dish observation of a pulsar whose radiation is scattered by a single
+one-dimensional screen. It explains how to use the :py:mod:`screens.screen`
+module, in particular its :py:class:`~screens.screen.Screen1D` class and the
+:py:meth:`~screens.screen.Screen.observe` method, to quickly set up a
+one-dimensional scattering screen and generate synthetic observations of the
+pulsar through such a screen.
+
+The combined codeblocks in this tutorial can be downloaded as a Python script
+and as a Jupyter notebook:
+
+:Python script:
+    :jupyter-download:script:`screen1d.py <screen1d>`
+:Jupyter notebook:
+    :jupyter-download:notebook:`screen1d.ipynb <screen1d>`
+
+
+Preliminaries
+=============
+
+Imports.
+
+.. jupyter-execute::
+
+    import numpy as np
+    import matplotlib.pyplot as plt
+    from matplotlib.colors import LogNorm
+
+    from astropy import units as u
+    from astropy.coordinates import (
+        CartesianRepresentation, CylindricalRepresentation,
+        UnitSphericalRepresentation
+    )
```

Comment:
Prefer it at the end of the line. Keeps the python space-defines-meaning idea, rather than the C curly-brackets-define-meaning